corehq/apps/accounting/views.py

```diff
@@ -1438,7 +1438,8 @@ def paginated_list(self):
                     "(domain: %(domain)s), but no billing record!" % {
                         'invoice_id': invoice.id,
                         'domain': self.domain,
-                    }
+                    },
+                    show_stack_trace=True
```

Comment:
nit, is this missing the trailing comma? Did I just out stickler stickler?

Reply:
Do you mean the ```show_stack_trace=True``` part?